examples/community-detection.ipynb

```diff
@@ -0,0 +1,1148 @@
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df2 = gds.louvain.mutate(Largest_CC, mutateProperty='louvainCommunityId')\n",
+    "df2"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "7563e824",
+   "metadata": {},
+   "source": [
+    "We get a modularity score of 0.5898 for our community detection algorithm."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "46969ec5",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.5898798012505129"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df2.modularity"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "15fc2baa",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Subreddit    [componentId, louvainCommunityId]\n",
+       "dtype: object"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Largest_CC.node_properties()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "858c65be",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>communityCount</th>\n",
+       "      <th>modularity</th>\n",
+       "      <th>modularities</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>300</td>\n",
+       "      <td>0.58988</td>\n",
+       "      <td>[0.4494089141198883, 0.5373675216145954, 0.555...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   communityCount  modularity  \\\n",
+       "0             300     0.58988   \n",
+       "\n",
+       "                                        modularities  \n",
+       "0  [0.4494089141198883, 0.5373675216145954, 0.555...  "
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "query = \"\"\"\n",
+    "    CALL gds.louvain.write('largest_connected_components2', { writeProperty: 'louvainCommunityId' })\n",
```

Comment:
Similar to above, you already ran `mutate` so you can write back the mutated property directly instead of running louvain again.

Reply:
Also you dont need to use `run_query` here but can use `gds.graph.nodeProperties.write(Largest_CC, ["louvainCommunityId"])`

Reply:
`run_query` wasn't working for me. Said it was not a valid function or so hence I used `run_cypher` instead.

Reply:
Absolutely `run_cypher` is the correct name of the method :+1: